src/test/java/io/dashboardhub/pipelinedashboard/definitions/ProfileDefinitions.java

```diff
@@ -1,7 +1,61 @@
 package io.dashboardhub.pipelinedashboard.definitions;
 
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
 import io.dashboardhub.pipelinedashboard.PipelinedashboardApplicationTests;
+import io.dashboardhub.pipelinedashboard.domain.User;
+import io.dashboardhub.pipelinedashboard.repository.UserRepository;
+import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.history.Revision;
+
+import java.lang.reflect.Field;
+import java.util.List;
 
 public final class ProfileDefinitions extends PipelinedashboardApplicationTests {
 
+    @Autowired
+    private UserRepository userRepository;
+
+    @Then("^there is a new revision for my User$")
+    public void there_is_a_new_audit_entry_for_my_user() throws Exception {
+        User currentUser = userRepository.findByUsername("TestUser");
+        Revision<Integer, User> latestRevision = userRepository.findLastChangeRevision(currentUser.getId());
+        Assert.assertTrue(latestRevision.getRevisionNumber() > 1);
+    }
+
+    @And("^the latest revised value for (.*) is (.*)$")
+    public void the_latest_revised_value_for(String fieldName, String value) throws Exception {
+        checkRevisionFieldValue(fieldName, value, 0);
+    }
+
+    @And("^the previous revised value for (.*) is (.*)$")
```

Comment:
If under `Then` section, should be `@Then`. `And` is to generic and could be used in any section.